### PR TITLE
KOGITO-1092 Include Kogito Data Index and Management Console as part …

### PR DESCRIPTION
**Pull request by cristianonicolai**
…of docker compose services

### CODE REVIEW

**Comment by danielezonca** [comment]
What about write also an integration test that uses that `docker-compose` file? Similar to [this test](https://github.com/kiegroup/kogito-examples/blob/master/dmn-drools-quarkus-metrics/src/test/java/org/kie/kogito/examples/GrafanaDockerComposeIT.java#L40).

**Comment by cristianonicolai** [comment]
@danielezonca I dont see much benefit in doing a test, as opposed to the docker compose you showed, Im only using already published images, not actually using anything that was build as part of the project.

**Review by nmirasch — APPROVED** [review]
works fine, really easy to see the services running

**Comment by danielezonca** [comment]
> @danielezonca I dont see much benefit in doing a test, as opposed to the docker compose you showed, Im only using already published images, not actually using anything that was build as part of the project.

Yes I agree that it is less useful. I think the main benefit is to make sure the script is still working, for example if we add a Param to data index or we change the path of persistence file.
Btw it is not mandatory :)

**Comment by cristianonicolai** [comment]
jenkins please retest this

**Review by pefernan — APPROVED** [review]
+1 Thanks!

**Comment by pefernan** [comment]
jenkins retest this please

File: kogito-travel-agency/extended/docker-compose/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+    image: quay.io/kiegroup/kogito-data-index-infinispan:${KOGITO_VERSION}
+    ports:
+      - "8180:8080"
+    depends_on:
+      kafka:
+        condition: service_started
+      infinispan:
+        condition: service_healthy
+    volumes:
+      - ../scripts/persistence/:/home/kogito/data/protobufs/
```
Comment (pefernan) [inline kogito-travel-agency/extended/docker-compose/docker-compose.yml]: @cristianonicolai hey! just noticed that on the script you are defining the `PERSISTENCE_FOLDER` as`PERSISTENCE_FOLDER=target/classes/META-INF/resources/persistence/protobuf` so the protos aren't stored in the `../scripts/persistence` folder. Shouldn't you change this volume?

**Review by pefernan — CHANGES_REQUESTED** [review]

File: kogito-travel-agency/extended/docker-compose/docker-compose.yml (hunk 1 above)
Comment (cristianonicolai) [reply kogito-travel-agency/extended/docker-compose/docker-compose.yml]: good catch :)

**Comment by cristianonicolai** [comment]
jenkins please retest this

**Review by AjayJagan — APPROVED** [review]
Thanks a lot @cristianonicolai :+1:

File: kogito-travel-agency/extended/docker-compose/prometheus/prometheus.yml [hunk 2]
```diff
@@ -12,7 +12,7 @@ scrape_configs:
     #   password: 'kieserver1!'
 
     static_configs:
-      - targets: ['localhost:8080']
+      - targets: ['172.17.0.1:8080']
```
Comment (radtriste) [inline kogito-travel-agency/extended/docker-compose/prometheus/prometheus.yml:15]: where does this IP come from ?

Comment (cristianonicolai) [reply kogito-travel-agency/extended/docker-compose/prometheus/prometheus.yml:15]: docker internal/default network

**Review by radtriste — APPROVED** [review]

**Review by pefernan — APPROVED** [review]
+1

**Pull request merged**